src/constants.js

```diff
@@ -13,3 +14,15 @@ export const FPTI_TRANSITION = {
 export const FPTI_STATE = {
   PXP: ("PXP_CHECK": "PXP_CHECK"),
 };
+
+export const FRAUDNET_URL = {
```

Comment:
this is fine for our deadline but we should create a jira ticket to move these urls to csnw. I can help with the details